packages/ng/department/service/department-v4.service.ts

```diff
@@ -0,0 +1,51 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { ILuApiResponse, LuApiV3Service } from '@lucca-front/ng/api';
+import { ILuTree } from '@lucca-front/ng/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { ILuDepartment } from '../department.model';
+import { ILuDepartmentService } from './department-service.model';
+import { IApiDepartment } from './department-v3.service';
+
+@Injectable()
+export class LuDepartmentService extends LuApiV3Service<ILuDepartment> implements ILuDepartmentService<ILuDepartment> {
```

Comment:
> extends LuApiV3Service<ILuDepartment> 

Seems weird. Maybe we could not inherit anything. Are `getById` / `getAll` methods are used or only `getTrees` ?

Reply:
It seems to be only `getTrees`, i'll remove the extends